activation/e2e/atx_merge_test.go

```diff
@@ -248,7 +248,7 @@ func Test_MarryAndMerge(t *testing.T) {
 	}
 
 	client := ae2e.NewTestPoetClient(2, poetCfg)
-	poetSvc := activation.NewPoetServiceWithClient(poetDb, client, poetCfg, logger)
+	poetSvc := activation.NewPoetServiceWithClient(poetDb, client, poetCfg, logger, 1)
```

Comment:
I'm not saying we _must_ change this, but just a side note, that's the case where using a normal param instead of a functional option hinders readability in the tests. There's no way to tell what does this `1` means short of looking up the definition of `NewPoetServiceWithClient`.
As an alternative, maybe we could use a named constant like `testTickSize` here?